core/merginapi.h

```diff
@@ -681,6 +695,23 @@ class MerginApi: public QObject
     bool validateAuth();
     void checkMerginVersion( QString apiVersion, bool serverSupportsSubscriptions, QString msg = QStringLiteral() );
 
+    /**
+    * Extracts string code of an error json. If its not json or value cannot be parsed, QString() is return;
+    * \param data Data received from mergin server on a request failed.
+    */
+    QString extractServerErrorCode( const QByteArray &data );
+    /**
+    * Extracts string value of an error json. If its not json or value cannot be parsed, QString() is return;
+    * \param data Data received from mergin server on a request failed.
+    * \param key Where should be a value from data
+    */
+    QString extractServerErrorStringValue( const QByteArray &data, const QString &key );
```

Comment:
This function is now obsolete when we have the `extractServerErrorCode`, right? I vote for removing it ✌🏻